### PR TITLE
updates dockerfiles and generates gen-check

### PR DESCRIPTION
**Pull request by JalexChen**
since there was a change to cimg-shared, this updates the dockerfiles but does not publish

it should be noted that the dockerfiles in this PR will be reflected in the next release

edit: more importantly, the generated file gen-check allows for the cimg-orb to check the validity of the dockerfiles

### CODE REVIEW

File: 3.0/browsers/Dockerfile [hunk 1]
```diff
@@ -59,5 +31,33 @@ ENV DISPLAY=":99"
 RUN	printf '#!/bin/sh\nXvfb :99 -screen 0 1280x1024x24 &\nexec "$@"\n' | sudo tee /docker-entrypoint.sh && \
 	sudo chmod +x /docker-entrypoint.sh
 
+# Install a single version of Firefox. This isn't intended to be a regularly
+# updated thing. Instead, if this version of Firefox isn't want the end user
```
Comment (mattbrictson) [inline 3.0/browsers/Dockerfile:35]: Small typo 🙂 

```suggestion
# updated thing. Instead, if this version of Firefox isn't what the end user
```

File: 3.0/browsers/Dockerfile [hunk 2]
```diff
@@ -59,5 +31,33 @@ ENV DISPLAY=":99"
...
+	echo 'Pin: release n=focal' | sudo tee -a /etc/apt/preferences.d/firefox.pref && \
+	echo 'Pin-Priority: 500' | sudo tee -a /etc/apt/preferences.d/firefox.pref && \
+
+	sudo apt-get update && \
+	sudo apt-get install --no-install-recommends --yes firefox && \
+	sudo rm -rf /var/lib/apt/lists/* && \
+	firefox --version
+
+# Install a single version of Google Chrome Stable. This isn't intended to be a
+# regularly updated thing. Instead, if this version of Chrome isn't want the
```
Comment (mattbrictson) [inline 3.0/browsers/Dockerfile:53]:
```suggestion
# regularly updated thing. Instead, if this version of Chrome isn't what the
```

**Review by mattbrictson — COMMENTED** [review]
Hi! Just lurking here and spotted a couple typos. Hope this is helpful. 👋

File: 3.0/browsers/Dockerfile (hunk 1 above)
Comment (JalexChen) [reply 3.0/browsers/Dockerfile:35]: thanks @mattbrictson  - would you mind putting in a PR for https://github.com/CircleCI-Public/cimg-shared/blob/main/variants/browsers.Dockerfile.template and we can have you added in as a contributor?

**Review by felicianotech — APPROVED** [review]

**Pull request merged**